examples/01_forecasting.ipynb

```diff
@@ -613,11 +614,60 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 1.2.4 prediction intervals<a class=\"anchor\" id=\"section_1_2_4\"></a>\n",
+    "#### 1.2.4. multivariate forecasting <a class=\"anchor\" id=\"section_1_2_4\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Forecasters can also be used with multivariate datasets. Below is an example using the longley dataset from `sktime.datasets`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sktime.datasets import load_longley\n",
+    "from sktime.forecasting.compose import ColumnEnsembleForecaster\n",
+    "from sktime.forecasting.exp_smoothing import ExponentialSmoothing\n",
+    "from sktime.forecasting.trend import PolynomialTrendForecaster\n",
+    "\n",
+    "_, y = load_longley()\n",
+    "\n",
+    "y = y.drop(columns=[\"UNEMP\", \"ARMED\", \"POP\"])\n",
+    "\n",
+    "forecasters = [\n",
+    "    (\"trend\", PolynomialTrendForecaster(), 0),\n",
+    "    (\"ses\", ExponentialSmoothing(trend=\"add\"), 1),\n",
+    "]\n",
+    "\n",
+    "forecaster = ColumnEnsembleForecaster(forecasters=forecasters)\n",
+    "forecaster.fit(y, fh=[1, 2, 3])\n",
+    "\n",
+    "y_pred = forecaster.predict()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y_pred"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1.2.5 prediction intervals <a class=\"anchor\" id=\"section_1_2_5\"></a>\n",
     "\n",
-    "`sktime` provides a unified interface to return prediction interval when forecasting. This is possible directly in the `predict` function, by setting the `return_pred_int` argument to `True`. The `predict` method then returns a second argument, Not all forecasters are capable of returning prediction intervals, in which case an error will be raised.\n",
+    "`sktime` provides a unified interface to return prediction interval when forecasting. This is possible using the `predict_interval` function. Not all forecasters are capable of returning prediction intervals, in which case an error will be raised.\n",
```

Comment:
I would recommend to also talk about the following things:
* how to check whether a forecaster can make interval predictions
* the multi-index structure of the return
* that the input minus `coverage` is exactly as for `predict` (and how this relates to the other cases, workflows)
* that `coverage` can be a vector

Further things that I think you should look at:
* the argument is called `coverage`, not `alpha`. Please refer to `coverage` (single number) as "nominal coverage of the prediction interval" and say the interval is symmetric.
* please add `predict_quantiles`, it fits nicely together with `predict_interval`.

Reply:
- described how to check if a forecaster can make interval predictions
- described multi-index structire
- described how prediction quantiles and intervals are related
- explained that coverage can be a list of floats
- tried to replace alpha with coverage
- added predict_quantiles